src/components/AddBookingForm/AddBookingForm.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+import "./AddBookingForm.scss";
+
+const AddBookingForm = ({ bookings, setBookings }) => {
+  const [formData, setFormData] = useState({
+    id: "",
+    firstName: "",
+    surname: "",
+    email: "",
+    title: "",
+    roomId: "",
+    checkInDate: "",
+    checkOutDate: "",
+  });
+  const [nextId, setNextId] = useState(bookings.length + 1);
+
+  const addBooking = (newBooking) => {
+    setBookings([...bookings, newBooking]);
+  };
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormData((prevData) => ({ ...prevData, [name]: value }));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBooking = { ...formData, id: nextId };
+    addBooking(newBooking);
+    setFormData({
+      id: "",
+      firstName: "",
+      surname: "",
+      email: "",
+      title: "",
+      roomId: "",
+      checkInDate: "",
+      checkOutDate: "",
+    });
```

Comment:
This object initalization is done twice with the same data. It works, but when you see code repeated like this it's worth thinking about whether you could put it inside a function - makes for tider code :)